cmd/stack_lint_test.go

```diff
@@ -206,16 +221,19 @@ func Test_KILLValue(t *testing.T) {
 	}
 
 	output = strings.Join(lines, "\n")
-	err = ioutil.WriteFile("../cmd/testdata/test-stack/image/Dockerfile-stack", []byte(output), 0644)
+	err = ioutil.WriteFile(filepath.Join(cmdtest.TestDirPath, "test-stack", "image", "Dockerfile-stack"), []byte(output), 0644)
 
 	if err != nil {
 		t.Fatal(err)
```

Comment:
These tests will fail if they are run in parallel, since they all write and read from the same file. I ran `go test` locally and it failed. It doesn't fail in travis since this runs `make test` which passes `--parallel 1`. A way to solve this would be to copy the `testdata` directory in the `setupSandbox` function.

Reply:
Yes - we could do this as part of a different PR.